code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -215,11 +220,12 @@
 
 /obj/item/reagent_containers/hypospray/medipen/survival
 	name = "survival medipen"

```

Comment:
i'd suggest changing the name to something like low-pressure survival injector to avoid confusion

Reply:
I already stated the low pressure thing in the description. It should be enough?